src/styles/lines/lines.js

```diff
@@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
 import Geo from '../../geo';
 import Utils from '../../utils/utils';
 
+let fs = require('fs');
+let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
```

Comment:
Makes sense. It might actually make sense for `Lines` to inherit from
`Polygons`, but that's outside the scope of this branch and we can revisit
later to consider that.

On Mon, Apr 18, 2016 at 11:51 AM, David Valdman <notifications@github.com>
wrote:

> In src/styles/lines/lines.js
> <https://github.com/tangrams/tangram/pull/291#discussion_r60083404>:
>
> > @@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
> >  import Geo from '../../geo';
> >  import Utils from '../../utils/utils';
> >
> > +let fs = require('fs');
> > +let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
>
> Yeah. It's more of a bikeshedding philosophical choice, as lines
> shouldn't depend on polygons so it shouldn't import it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram/pull/291/files/b579388894cbe6dcec7340df73ca7453ae1135ba#r60083404>
>